doc/plugins/hmac-auth.md

```diff
@@ -127,7 +128,10 @@ x-custom-header:value
 ### Use the generated signature to try the request
 
 **Note: ACCESS_KEY, SIGNATURE, ALGORITHM, DATE, SIGNED_HEADERS respectively represent the corresponding variables**
-**Note: SIGNED_HEADERS is the headers specified by the client to join the encryption calculation**
+
+**Note: SIGNED_HEADERS is the headers specified by the client to join the encryption calculation. If there are multiple headers, they must be separated by ";": `x-custom-header-a;x-custom-header-b`**
+
+**Note: SIGNATURE needs to use base64 for encryption: `base64_encode(SIGNATURE)`**
 
```

Comment:
it would be better to use one `Note` here.
e.g :
Note:

1. xxx

1. xxx

Reply:
Thanks, I will modify the PR